`python/tests/test_sbml_import.py::test_import_same_model_name` fails with swig 4.4.0

`python/tests/test_sbml_import.py::test_import_same_model_name` fails with swig 4.4.0:

```
>       with pytest.raises(RuntimeError, match="in the same location"):
E       Failed: DID NOT RAISE <class 'RuntimeError'>

python/tests/test_sbml_import.py:1001: Failed
```

SWIG 4.4.0 was [released](https://github.com/swig/swig/releases/tag/v4.4.0) last week and is [available on PyPI](https://pypi.org/project/swig/4.4.0/) since yesterday.

last successful run: https://github.com/AMICI-dev/AMICI/actions/runs/18830244876/job/53720321534
first failure: https://github.com/AMICI-dev/AMICI/actions/runs/18864473329/job/53829395293

--- a/swig/modelname.template.i
+++ b/swig/modelname.template.i
@@ -57,7 +57,7 @@ using namespace amici;
 // store the time a module was imported
 %{
 #include <chrono>
-static std::chrono::time_point<std::chrono::system_clock> _module_import_time;
+static std::chrono::time_point<std::chrono::system_clock> _module_import_time = std::chrono::system_clock::now();
 
 static double _get_import_time() {
     auto epoch = _module_import_time.time_since_epoch();
@@ -68,7 +68,9 @@ static double _get_import_time() {
 static double _get_import_time();
 
 %init %{
-    _module_import_time = std::chrono::system_clock::now();
+    // NOTE: from SWIG 4.4.0 onwards, %init code is executed every time the
+    // module is executed - not only on first import
+    // This code ends up in `SWIG_mod_exec`.
 %}
 
 

